app/main/views/briefs.py

```diff
@@ -162,6 +162,7 @@ def list_briefs():
 
 @main.route('/briefs/<int:brief_id>/status', methods=['PUT'])
 def update_brief_status(brief_id):
+    """Route is deprecated. Use `update_brief_status_by_action` instead."""
```

Comment:
Wonder if we should log a warning message in cases like this so an administrator can know if an old interface is still being used.